rpa-builder/6.5/modules/ROOT/pages/toolbox-variable-handling-combine-strings.adoc

```diff
@@ -1,77 +1,78 @@
+= Combine Strings
 
+Use *Combine Strings* to combine multiple variables into a single new variable. This action step supports Script variables, environment
+variables, and Pin variables. You can insert text before, after, and between variables.

```

Comment:
Script variables and Pin variables are using upper case. Should environment variables follow this?